test/runtime.js

```diff
@@ -173,22 +206,44 @@ describe('runtime parameters', function() {
         assert.equal(process.sass.binaryUrl.substr(0, URL.length), URL);
       });
     });
-    describe('checking if environment variable overrides package.json', function() {
+    describe('checking if environment variable overrides .npmrc configuration variable', function() {
```

Comment:
You're right, this is a much better solution. I will start refactoring all configuration unit tests.